aiohttp/web_runner.py

```diff
@@ -58,7 +57,9 @@ def name(self) -> str:
             self._runner._unreg_site(self)
             return  # not started yet
         self._server.close()
-        await self._server.wait_closed()
+        # named pipes do not have wait_closed property
```

Comment:
So why don't you create one?

Reply:
Named pipes in asyncio are from the class: https://github.com/python/cpython/blob/master/Lib/asyncio/windows_events.py#L241

Which comprises of only normal sync methods. `.close()` is a sync method as well. So therefore it has a `.closed()` property. Are you suggesting to add `.wait_closed()` property there for consistency(hence a PR to cython)?
Or is it something else?

Reply:
Yeah, I'd just inherit that and return immediately maybe. Let's see what @asvetlov thinks.